app/controllers/todo_lists_controller.rb

```diff
@@ -13,16 +13,27 @@ def new
 		@todo_list = TodoList.new
 	end
 
+	def edit
+		@todo_list = TodoList.find(params[:id])
```

Comment:
The indentation is too much, let's move to using 2 spaces for each tab.

Reply:
I tried hard